services/web/client/source/class/osparc/utils/Utils.js

```diff
@@ -366,7 +366,7 @@ qx.Class.define("osparc.utils.Utils", {
         return "0 Bytes";
       }
       const i = parseInt(Math.floor(Math.log(bytes) / Math.log(1024)));
-      return Math.round(bytes / Math.pow(1024, i), 2) + " " + sizes[i];
+      return Math.round((bytes / Math.pow(1024, i)) * 100) / 100 + " " + sizes[i];
     },
 
     bytesToGB: function(bytes) {
```

Comment:
I guess: you don't need this function anymore?

Reply:
The clusters view still uses it, I guess I could merge them at some point